src/module.c

```diff
@@ -413,7 +413,8 @@ int DropIndexCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
       RMUtil_StringEqualsCaseC(argv[0], "_FT.DROP")) {
     RedisModule_Replicate(ctx, RS_DROP_IF_X_CMD, "v", argv + 1, argc - 1);
```

Comment:
I think we should handle it here as well

Reply:
@MeirShpilraien  `FT.DROP` is deprecated on RS 2.0. Should we fix it anyway?

Reply:
Sorry you are right :+1:

Reply:
Although I do think we should fix this as well, we do not want someone to do it and crash the server..

Reply:
Done